<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSystemWindowSkin</class>
 <widget class="QDialog" name="DialogSystemWindowSkin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window skin...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
         <item>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>593</width>
              <height>362</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="WidgetShowPicture" name="widgetShowPicture" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>108</width>
                   <height>288</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="0" colspan="2">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>558</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxZoom">
           <property name="title">
            <string>Zoom</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QSlider" name="horizontalSlider">
              <property name="minimum">
               <number>-5</number>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
              <property name="pageStep">
               <number>1</number>
              </property>
              <property name="sliderPosition">
               <number>0</number>
              </property>
              <property name="tracking">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="invertedControls">
               <bool>false</bool>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBelow</enum>
              </property>
              <property name="tickInterval">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-460</y>
            <width>334</width>
            <height>894</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="labelPicture">
               <property name="text">
                <string>Picture:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="WidgetPicture" name="widgetPicture" native="true">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>27</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>27</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBoxSideBorders">
               <property name="title">
                <string>Side borders</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectTopLeft" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectTopRight" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectBotLeft" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectBotRight" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBoxBorders">
               <property name="title">
                <string>Borders</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectLeft" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectRight" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectTop" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectBot" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBoxBackground">
               <property name="title">
                <string>Background</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectBackground" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectBackgroundSelection" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
                    <item>
                     <widget class="QLabel" name="labelOption">
                      <property name="text">
                       <string>Option:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="comboBoxOptionBackground"/>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>98</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBoxArrows">
               <property name="title">
                <string>Arrows</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectArrowTargetSelection" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectArrowEndMessage" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectArrowUpDown" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBoxTexts">
               <property name="title">
                <string>Texts</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectTextNormal" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectTextHeal" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectTextCritical" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="WidgetSelectRect" name="widgetSelectRectTextMiss" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetPicture</class>
   <extends>QWidget</extends>
   <header>widgetpicture.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetShowPicture</class>
   <extends>QWidget</extends>
   <header>widgetshowpicture.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetSelectRect</class>
   <extends>QWidget</extends>
   <header>widgetselectrect.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSystemWindowSkin</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSystemWindowSkin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
